src/v0/destinations/hs/util.js

```diff
@@ -506,6 +509,7 @@ const getExistingData = async (inputs, destination) => {
 
   while (checkAfter) {
     const endpoint = IDENTIFY_CRM_SEARCH_ALL_OBJECTS.replace(':objectType', objectType);
+    const endpointPath = `objects/""/search`;
```

Comment:
For readability
```suggestion
    const endpointPath = `objects/"<objectType>"/search`;
```

Reply:
We were avoiding any dynamic values in our endpointPath label so as to get a general count of the endpoint else here for each objectType (which will be created from externalId as HS-"some-object-name" we will have a value and hence the metric will be segregated for all such labels rather than a unified one

Reply:
**Point I think @krishna2020 is trying to make**
The endpoint actually might contain some dynamic values in endpoint but the endpoint string we send to metrics doesn't contain details of the dynamic-value
There is a possibility for us to assume that the endpoint in the metric is exactly same as what we send to destination(transformation/proxy).
Hence having a value-indicator in endpoint string we send to metrics is helpful for us

Please correct me if I'm wrong @krishna2020

Reply:
Yes @sanpj2292 
endpoint string we send to metrics doesn't contain details of the dynamic-value

So just having the placeholder of the variable name instead of "" helps in readability